sdk/eventhub/tests.yml

```diff
@@ -9,15 +9,19 @@ jobs:
         Linux_Python35:
           OSVmImage: 'ubuntu-18.04'
           PythonVersion: '3.5'
+          CoverageArg: '--disablecov'
         Linux_Python39:
           OSVmImage: 'ubuntu-18.04'
           PythonVersion: '3.9.0'
+          CoverageArg: ''
         Windows_Python27:
           OSVmImage: 'windows-2019'
           PythonVersion: '2.7'
+          CoverageArg: '--disablecov'
         MacOs_Python37:
           OSVmImage: 'macOS-10.15'
           PythonVersion: '3.7'
+          CoverageArg: '--disablecov'
```

Comment:
```suggestion
          CoverageArg: ''
```

Reply:
We can chase down owners, but in the meantime, let's get it publishing coverage in general.